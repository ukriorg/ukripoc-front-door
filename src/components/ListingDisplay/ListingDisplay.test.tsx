import React from "react";
import renderer from "react-test-renderer";
import { ListingDisplay } from "./index";

describe("ListingDisplay", () => {
    it("renders correctly", () => {
        const tree = renderer.create(<ListingDisplay />).toJSON();
        expect(tree).toMatchSnapshot();
    })
});

