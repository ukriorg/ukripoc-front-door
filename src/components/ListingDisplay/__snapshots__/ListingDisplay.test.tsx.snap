// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListingDisplay renders correctly 1`] = `
<section>
  <a
    aria-current="page"
    className="sc-bdVaJa cQDaMj"
    href="/"
    onClick={[Function]}
  >
    Back to all opportunities
  </a>
</section>
`;
